Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -344,6 +344,16 @@ def custom_callback(name, progress):
     cast = image.astype(np.float32)
     assert cast.dtype == np.float32
 
+    # Test .astype for conversion between vector-like pixel types.
+    components = 3
+    numpyImage = np.random.randint(0, 256, (12,8,components)).astype(np.uint8)
+    input_image = itk.image_from_array(numpyImage, is_vector=True)
+    if (type(input_image) == itk.Image[itk.RGBPixel[itk.UC],2] and
+        hasattr(itk.CastImageFilter, 'IRGBUC2IVF32')):
+        output_pixel_type = itk.Vector[itk.F,components]
+        output_image = input_image.astype(output_pixel_type)
+        assert type(output_image) == itk.Image[output_pixel_type, 2]
+
 except ImportError:
```

Comment:
Unrelated to this PR, we should remove this exception. We now require `numpy`.

Reply:
I think you are talking about numpy's `astype` -- have I got that right? -- but this is itk's `astype`.

Reply:
No. There is `import numpy` statement [here](https://github.com/InsightSoftwareConsortium/ITK/pull/2085/files/8f2a6c99fdb76d36abe1054dc29acc576b533371#diff-2fbfe1452d1a1c098ebb20ccb96106c42c6c3b84b9eac15449eaa6958d8ef269L237). If it fails, `except ImportError` [here](https://github.com/InsightSoftwareConsortium/ITK/pull/2085/files/8f2a6c99fdb76d36abe1054dc29acc576b533371#diff-2fbfe1452d1a1c098ebb20ccb96106c42c6c3b84b9eac15449eaa6958d8ef269L347-L350) will silently ignore it. Since we now explicitly require `numpy`, we should complain, not ignore. But better left to a different PR.

Reply:
Got it.  Thanks!